lib/active_admin.rb

```diff
@@ -6,6 +6,7 @@
 require 'bourbon'
 require 'kaminari'
 require 'formtastic'
+require 'formtastic_i18n'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.